about.html

```diff
@@ -0,0 +1,147 @@
+           CELEBRATING THEIR WORKS AND CONNECTING THEM TO OPPORTUNITIES FOR 
+           LEARNING, .
+          COLLABORATION AND FUNDING..
+        </h4>
+        <p class="gamescom-p">
+          GAMESCOM is a platform that brings together enthusiasts and digital creatives across the games, animation, comics, 
+          and movie industry in America via insightful events such as conventions, game jams and so much more.
+          Our aim is to invite investments into the space and foster collaboration among digital creatives across America as well as celebrating the rich and amazing level of creativity possessed by Americans.
+        </p>
+        <button type="button" class="recap">2020 RECAP</button>
+        <div class="about-email">
+          <p>
+            Please contact us on this Email for any further questions or 
+            suggestions about GAMESCOM event 2021!
+          </p>
+          <p><a href="">gamescom@gamemail.com</a></p>
+        </div>
+      </div>
+    </section>
+    <section class="past-gamescom">
+      <h2>View past GAMESCOM Events</h2>
+      <hr>
+      <p>Take a look at the last two GAMESCOM Events which took place in
+         America and Spain
+      </p>
+      <div class="last-events">
+        <div class="events-container">
+          <div class="events">
+            <h3>2019</h3>
+            <p>GAMESCOM Event 2019 in America</p>
+          </div>
+        </div>
+        <div class="events-container">
+          <div class="events">
+            <h3>2019</h3>
+            <p>GAMESCOM Event 2020 in Spain</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <hr>
+    <section class="partners">
+      <div class="partner-text">
+        <h2>PARTNERS</h2>
+      </div>
+      <hr class="red-line">
+      <div class="partner-card-wrap">
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/squere.jpg" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/capcom.jpg" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/infinity.jpg" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/sega.jpg" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/bethesda-2.png" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/bungie.jpg" alt="">
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Please delete this section because in the original design there is no partners section. 

![Capture](https://user-images.githubusercontent.com/64173189/142464288-4ecf9a83-1490-4733-838d-182d0e32b4a5.JPG)